lib/fluent/env_utils.rb

```diff
@@ -0,0 +1,50 @@
+require 'fluent/config'
+
+module Fluent
+  module EnvUtils
+    def self.convert_env_var_to_type(value, type)
+      Fluent::Config.reformatted_value(type, value)
+    end
+
+    class OjOptions
+      OJ_OPTIONS = {
+        'bigdecimal_load': :symbol,
+        'max_nesting': :integer,
+        'mode': :symbol,
+        'use_to_json': :bool
+      }
+  
+      OJ_OPTIONS_ALLOWED_VALUES = {
+        'bigdecimal_load': %i[bigdecimal float auto],
+        'mode': %i[strict null compat json rails object custom]
+      }
+  
+      OJ_OPTIONS_DEFAULTS = {
+        'bigdecimal_load': :float,
+        'mode': :compat,
+        'use_to_json': true
+      }
+
+      def initialize
+        @options = {}
+        OJ_OPTIONS_DEFAULTS.each { |key, value| @options[key] = value }
+      end
+
+      def get_options
```

Comment:
`get_options` may be a redundant name.
It seems that `options` is enough.

Reply:
Would that not clash with `@options` above?

Reply:
It seems that it does not conflict with it.